### ISSUE TITLE
piano roll

### ISSUE THREAD

**Issue by qulf** [elieserdejesus/JamTaba#1142]
hi.suggestion for standalone version. mayby somebady mention before. midi lopper with edit function. to be able to play midi loops from midi files or record midi loops. somethings like piano roll, midi editor for looper. another things. midi input from computer keyboard. :) i am not keyboard player but when i tryed to play some simple drum beat from laptop i stuck. thanks .cy on jam.

**Comment by qulf** [comment]
Or maybe someone knows vst plugin with such possibilities? Piano roll vst?

**Comment by qulf** [comment]
Yes but i speak about standalone version of jamtaba. i like to use sometimes that wersion. She ( jamtaba is a woman?!?) is lighter and not  like big,fat and heavy DAWs.

**Comment by fab672000** [comment]
I actually use it in standalone mode a lot too, could be fun to develop DAW like editing features, maybe we could reuse some public code there instead of re-inventing it?

**Comment by elieserdejesus** [comment]
> maybe we could reuse some public code there instead of re-inventing it

It's ok for me! 👍

**Comment by qulf** [comment]
Maybe not DAW-software but advanced looper. The market is full of DAWs. Something like a mini sequencer connected to the looper. You decide

**Comment by qulf** [comment]
https://www.image-line.com/flstudio/history.php?entry_id=1286984399&title=fl-studio-10
